packages/cardano-services/src/Handle/TypeOrmHandleProvider.ts

```diff
@@ -55,4 +56,16 @@ export class TypeOrmHandleProvider extends TypeormProvider implements HandleProv
       return handles.map((handle) => mapEntity(entities.find((entity) => entity.handle === handle)));
     });
   }
+
+  async getPolicyIds(): Promise<Cardano.PolicyId[]> {
+    return this.withDataSource(async (dataSource) => {
```

Comment:
If >1 policyId is found, we should cache it to not repeat the query for every request

Reply:
Fixed on 5b9717e

Reply:
@mkazlauskas won't this cache be invalid if a projector, configured with multiple policies, only has handles from one policy early in the chain history and the API is called?

Reply:
I'm OK to leave this query optimisation in place, but maybe we should consider maintaining a policy IDs meta table on projector startup as future work?

Reply:
Yes you're right @rhyslbw , but shouldn't be an issue for now. We could also just not cache this (or short duration cache) if not at the tip, but I think for initial implementation this is fine.

Reply:
LW-7725